<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Кассовый аппарат</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="table">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>661</width>
      <height>311</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Тип</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Время</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Сумма</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Название фонда</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnPush">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>89</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leAmount">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>410</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Сумма</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leFundName">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>222</x>
      <y>410</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Название фонда</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dteTime">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>370</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>dd.MM.yyyy H:mm:ss</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbPurchase">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>330</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Purchase</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbDonation">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>330</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Donation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPop">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>389</y>
      <width>89</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="time1">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>360</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>dd.MM.yyyy H:mm:ss</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="time2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>360</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>dd.MM.yyyy H:mm:ss</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCalc">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>400</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Расчитать выручку</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTime1">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>330</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Начало</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTime2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>330</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Конец</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="menuOpen"/>
    <addaction name="menuSave"/>
    <addaction name="separator"/>
    <addaction name="menuQuit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Кассовый апарат</string>
    </property>
    <addaction name="menuDeleteAll"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="menuQuit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="menuOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="menuSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="menuDeleteAll">
   <property name="text">
    <string>Удалить всё</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
